client/src/App.js

```diff
@@ -1,15 +1,15 @@
 import {CssBaseline} from "@mui/material";
 import Router from "./router"
 import {ApolloClient, ApolloLink, ApolloProvider, from, HttpLink, InMemoryCache} from '@apollo/client';
-import {BASE_URL} from './config';
 import {AppContext} from "./providers/context/appContext";
 import {useContext} from "react";
-import {I18nProvider, LOCALES} from "./providers/i18n";
+import {I18nProvider} from "./providers/i18n";
 
 const App = () => {
     const {state} = useContext(AppContext)
 
-    const httpLink = new HttpLink({uri: BASE_URL});
+    // const httpLink = new HttpLink({uri: `${location.origin}/graphql`});
```

Comment:
seems like you forgot to remove your debug code

Reply:
will be removed in next task

Reply:
Then add todo

Reply:
will be used before deployment to host

Reply:
fixed in 609 task